src/build/mod.rs

```diff
@@ -7,7 +7,7 @@ pub mod arg;
 mod arg_group;
 mod usage_parser;
 
-pub use self::app::{App, AppFlags, AppSettings, Propagation};
-pub use self::arg::{Arg, ArgFlags, ArgSettings};
```

Comment:
This is not related to this PR.

Reply:
Removing unneeded `pub`s (including reexports) directly relate to this PR, see teh rationale in the comment above